docs/quickstart-aws.md

```diff
@@ -179,24 +180,34 @@ kubectl --kubeconfig=<cluster_name>-kubeconfig
 ```
 
 or export the `KUBECONFIG` variable environment variable:
+
 ```bash
 export KUBECONFIG=$PWD/<cluster_name>-kubeconfig
 ```
 
 ## Scaling Worker Nodes
 
-As worker nodes are managed by machine-controller, they can be scaled up and down
-(including to 0) using Kubernetes API.
+Wrker nodes are managed by the machine-controller. It creates initially one per
```

Comment:
```suggestion
Worker nodes are managed by the machine-controller. It creates initially one per
```

and all other "Wrker" in other files

Reply:
Hmmpf, should review my own changes. 🤷‍♂